support-frontend/assets/components/thankYou/thankYouModule.tsx

```diff
@@ -253,21 +244,22 @@ function ThankYouModule({
 		trackComponentLoadId && trackComponentLoad(trackComponentLoadId);
 	}, []);
 
-	const isDownloadModule = moduleType === 'appDownload';
+	const hasQrCodes = moduleType === 'appDownload';
 	const isDownloadModules = moduleType === 'appsDownload';
+	const hasImagery = ['appDownload', 'newspaperArchiveBenefit'].includes(
```

Comment:
I've reused the imagery layout but I think we should just delete `appDownload` as an option (separate PR to keep it small) since it's unused